tests/Microsoft.Identity.Test.Unit/ParallelRequestsTests.cs

```diff
@@ -85,6 +85,138 @@ public async Task ExtraQP()
             Assert.AreEqual(NumberOfRequests, results.Length);
         }
 
+        [TestMethod]
```

Comment:
Verifies concurrency behavior when acquiring tokens for different tenant IDs in parallel.